src/v2/Apps/BuyerGuarantee/Routes/BuyerGuaranteeIndex.tsx

```diff
@@ -373,7 +377,7 @@ export const BuyerGuaranteeIndex: React.FC<BuyerGuaranteeIndexProps> = ({
           <Image
             src={securePaymentImage.image.resized.url}
             alt={securePaymentImage.artist.name}
-            lazyLoad
+            loading="lazy"
```

Comment:
So actually: you'd prefer the `lazyLoad` prop here — the only reason I use the native one in the header image is because we are unable to use this component inside of a picture tag (right now).